admin/class-yoast-notification.php

```diff
@@ -110,12 +110,12 @@ public function get_id() {
 	}
 
 	/**
-	 * Retrieve the user to show the notification for.
+	 * Retrieve the id of the user to show the notification for.
 	 *
-	 * @return WP_User The user to show this notification for.
+	 * @return int The user id.
 	 */
 	public function get_user() {
```

Comment:
Changing the return value and type is a BC break. I think this is actually just a helper for the method below. 

Probably we can just deprecate it?
But perhaps it is wise to not alter it and retrieve the user after getting the ID instead (similar to your `has_capability` change)? For BC sake.

Reply:
I deprecated it, it wouldn't make sense anymore after this PR gets merged 🙂